soccer/MainWindow.cpp

```diff
@@ -318,6 +318,14 @@ void MainWindow::updateViews()
 
 	if (currentFrame)
 	{
+		if(_firstLogTimestamp == -1)
+			_firstLogTimestamp = currentFrame->timestamp();
+		auto gametime_ms = (currentFrame->timestamp()-_firstLogTimestamp)/1000;
```

Comment:
These are just ints/longs right? 
Can we just use the actual type if its not a really complicated template so can tell what they are easily?
Or are we using these for some reason like the types of the timestamp could change?